### DISCUSSION TITLE
ROS2 support

### DISCUSSION THREAD

**Discussion by macmacal** [WPI-AIM/ambf#193] [Q&A]
Hello,

are there any plans for supporting ROS2 in AMBF?
I was looking for any previous discussion about it, but I didn't find anything related to such question.

Best regards
Maciej

**Comment by adnanmunawar** [comment]
Hello Maciej,

That is correct, we have not yet implemented any ROS 2 modules. If there is adequate interest, we can look towards implementing it in a couple of weeks when the academic year begins to cool down.
